chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/AddedToRoomTest.kt

```diff
@@ -0,0 +1,103 @@
+package com.pusher.chatkit.rooms.state
+
+import assertk.assertThat
+import assertk.assertions.containsOnly
+import assertk.assertions.isNotNull
+import com.pusher.chatkit.state.ChatkitState
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class AddedToRoomTest : Spek({
+
+    describe("Given an initial state of empty rooms") {
```

Comment:
for consistency and simplicity it would be better to start from lower-case, it's really not a proper sentence (many thens and the lack of dot)

Reply:
👍 i refactored all the tests to be like your JoinedRoomsReceivedTests